<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>721</width>
     <height>481</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Aerolínea</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>200</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>26</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Sistema de viajes </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>images.png</pixmap>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>MatrizAdyacencia</string>
    </attribute>
    <widget class="QTableView" name="tableWidgetA">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>411</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>260</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Tamaño matriz</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="NumeroDeFilasSpinBoxA">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrepararTabla">
     <property name="geometry">
      <rect>
       <x>554</x>
       <y>80</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Preparar tabla</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnInsertarRegistro">
     <property name="geometry">
      <rect>
       <x>554</x>
       <y>110</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Insertar Ciudad</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBorrarSeleccion">
     <property name="geometry">
      <rect>
       <x>554</x>
       <y>140</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrar selección</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnBorrarTodo">
     <property name="geometry">
      <rect>
       <x>554</x>
       <y>170</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Borrar Todo</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="MatrizAdyacencia">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>111</width>
       <height>221</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditCiudad">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonAgregar">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>280</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Registrar</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnFloyd">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Floyd</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnDijkstra">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Dijkstra</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnPrim">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Prim</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnKruskal">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>370</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Kruskal</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Grafo</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
